### PR TITLE
Fix missing links for eMMC installation

### LINKED ISSUE #245: Restore simpler doc for emmc installation
The [previous doc](https://github.com/hexdump0815/imagebuilder/blob/3817d61b30069fe4541dac8612453ee34a597226/doc/install-to-emmc-on-arm-chromebooks.md) for installing to emmc was much easier to follow than the [current doc](https://github.com/hexdump0815/imagebuilder/blob/main/doc/install-to-emmc-with-luks-full-disk-encryption.txt).

I am unsure if it was deleted by mistake or if it intentional, but it would be great if it is brought back.

I looked at the current version, and was totally unsure which path I should take (there seems to be one option for general chromebooks, one for EFI, one for u-boot, one for arm64 chromebooks, and so one). While that may be useful for advanced users, as someone cloning USB image to emmc for the first time, the previous version (now deleted) came really handy.

---
I am running Debian Bookworm [turned into Raspberry Pi OS](https://github.com/hexdump0815/imagebuilder/blob/main/doc/install-to-emmc-with-luks-full-disk-encryption.txt) bookworm on a Chromebook HANA, and everything (except GPU acceleration) is working. While GPU acceleration does not work, I can still watch Youtube at decently high resolution and make video calls.

Thank you very much for maintaining this project.

### PR DESCRIPTION
**Pull request by archisman-panigrahi**
The links were not updated in first-steps.md, and I mistakenly assumed the tutorial has been deleted (#245).

Fixes #245

### CODE REVIEW

**Comment by archisman-panigrahi** [comment]
Can we replace basic-installation.md -> basic-emmc-installation.md? 
Then it would be easier to find by searching on GitHub.

If you agree, I can add that to this PR.

**Comment by thenameisluk** [comment]
github adds every commit you push to the pr branch to your pr
so it's already in the pr

you can force it back with some magic command i once was but i forgor

also not really a good idea
- it would be better to keep the name consistent with dualboot-instalation.md (still on the way) and luks-installation.md
- installation can per performed on another usb (and the luks one even makes a lot of sens to do that)
- there is no point in clarifying things that much, it could as well be named basic-velvet-os-onto-emmc-installation.md (over exaggeration)
- the name + the directory (doc/chromebook/...) already describes the purpose well enough

i appreciate the effort but i don't thinks it's a good idea to change the name of the file
still final word belongs to hexdump so will see

**Comment by archisman-panigrahi** [comment]
> so it's already in the pr

@thenameisluk No it's not. I did not rename any file in the second commit.
In the second commit I only changed 
`there are a few ways you could install system` --> `there are a few ways you could install to system storage (eMMC/SSD)`.

**Comment by archisman-panigrahi** [comment]
> you can force it back with some magic command i once saw but i forgor

The magic command is,
- Make changes
- `git add .`
- `git --amend -m "commit message`
- `git push --force`

But that is not necessary here because I did not rename the file.
It is totally OK if you don't want to rename the file. As long as the file is properly linked to the other docs, it is good.

The PR is done from my side.

**Comment by thenameisluk** [comment]
yeah that will conflict with the [pr 'm working on](https://github.com/thenameisluk/imagebuilder/blob/main/luk's%20special%20todo)

**Comment by archisman-panigrahi** [comment]
Did you mean, the current version of my PR (without any renaming) will conflict with your PR?

**Comment by thenameisluk** [comment]
i will move some stuff from regular installation to preparation one (the disclaimer part since it's relevant to all the installations)

should be an issue to resolve but just pointed that out

**Comment by archisman-panigrahi** [comment]
Ok. I can update this PR after you are done, then. It would be helpful if you can ping me once you are done.

**Comment by thenameisluk** [comment]
nah the pr can go through i can merge them on my side without any issue

rn 'm just [waiting for hexdump answer](https://github.com/hexdump0815/imagebuilder/issues/209#issuecomment-2372453696) before continuing with the rework

**Review by hexdump0815 — APPROVED** [review]
looks good to me

**Comment by hexdump0815** [comment]
@thenameisluk - sorry i just read that you have some changes in preparation in parallel, but looks like merging this here first seems to be no problem anyway as you said

**Pull request merged**
